lte/gateway/docker/s1ap/readme.md

```diff
@@ -71,3 +45,35 @@ If inside of container, CTRL+d or exit from container and run stop script
 root@472f8708ec12:/magma/lte/gateway/python/integ_tests# exit
 s1ap/stop-s1ap.sh
 ```
+
+# Simple example
+
+```
+sudo -i
+wget https://raw.githubusercontent.com/magma/magma/master/lte/gateway/deploy/agw_install_docker.sh
+mkdir -p /var/opt/magma/certs
+echo "-----BEGIN CERTIFICATE-----
+MIIDXzCCAkegAwIBAgIUakfCUNf7JMKbLDqHnuiG1QNhCQ8wDQYJKoZIhvcNAQEL
+BQAwPzELMAkGA1UEBhMCVVMxMDAuBgNVBAMMJ3Jvb3RjYS5tYWdtYS0xNi1zeWRu
+ZXkuZmFpbGVkd2l6YXJkLmRldjAeFw0yMTA3MTQyMjI5MjNaFw0zMTA3MTIyMjI5
+MjNaMD8xCzAJBgNVBAYTAlVTMTAwLgYDVQQDDCdyb290Y2EubWFnbWEtMTYtc3lk
+bmV5LmZhaWxlZHdpemFyZC5kZXYwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEK
+AoIBAQDFVmNFaAOkVD3c8W28FkGUVmBKDyj/T7N8C7PE43WvbBZJmO5TO1c887Dt
+yiX8Ua2mpCQ2SF8DZtXojkLGKOFM85uxzTV1YI656u5BDSejRkm1UDeMT5R+tQJK
+fyHYTt5ZNprX/dUrxYnp+h2zEl0PlzO5ijrktuZgM4KZjtQVaC1VirSC//2ZKQEo
+2aX3L81ALrjVzsmH4TePKEY8StjDHC2Mg6LOaYR/+Gu272P39/heULrm147g1k0k
+haeKv8qrI0dfvBcZveTzYf77iA6/OeVzYtWwM3zr1Z1cFALZrcuS6R6DrsAInseH
+qeiMh4kLfyoh0vQCNpEAJQgt5PmVAgMBAAGjUzBRMB0GA1UdDgQWBBTpk41oSZDv
+hSlsCLboVWzT5w414TAfBgNVHSMEGDAWgBTpk41oSZDvhSlsCLboVWzT5w414TAP
+BgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQBWeu3+kB2WmxXbIDPU
+1JGw4rvj/+u4mvN1maYkibqCZyKMXLuqwOy9wMhtniHgKp/RIxFI+W3FTq4Tik++
+kwDemaYq3nwbHMwBXwFh/T9I9ExtWBCogj+LLFUsrPDJNmUwYnnEMRh6beF8oT1E
+Da3oNVZ70Tyv0DnWozW+4TQZ8bTOQ/bpjoFNZPVB3Jr7tjVLfPez8m/clM8+War+
+gjTiiUsJkJP2uhKmWkb58CCiH+k2EH3fw2IUmc0fgTMGZ5vv8g1OjCBrXspnGSpk
+iJf9ryw/jIH/9RGxSUO7tiQxe/IShf65clsyxlAjrSr7JvbYwOyIXAbgNA7vk0lc
+nmmv
+-----END CERTIFICATE-----" > /var/opt/magma/certs/rootCA.pem
+bash agw_install_docker.sh
+sed -i 's/DOCKER_REGISTRY=/DOCKER_REGISTRY=public.ecr.aws\/yourrepo\//' /var/opt/magma/docker/.env
```

Comment:
It might be more sensible to use the local docker registry, which ships with your docker installation on Ubuntu 20.04 here. That way the images can be added to the local registry in multiple ways but this code is agnostic of the way you obtained the image.

Reply:
This is true, but I put this step here just to be clear that this is where you would put your registry values. Put in the doc that you can leave the registry value blank to use the local registry.